array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,12 @@
+/* exported compact */
+function compact(array) {
+
+  var newArray = [];
+
+  for (var i = 0; i < array.length; i++) {
+    if (Boolean(array[i]) !== false) {
```

Comment:
Nice work! another check you can do which will have the same results is:
```javascript
if(array[i])
```
this will work because if the current value of the array is truthy, the if statement will run

Reply:
Got it. Thank you!